cli/cmd/status.go

```diff
@@ -39,7 +39,8 @@ func internalStatusModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 	}
 
 	for _, run := range runningCtx.Instances {
-		log.Infof("%s: %s", run.InstName, running.Status(&run))
+		procStatus := running.Status(&run)
+		log.Infof("%c %s: %s", procStatus.Color, run.InstName, procStatus.Text)
```

Comment:
Maybe we should just use the colorful output using a standard [package color](https://pkg.go.dev/github.com/fatih/color)?

Reply:
This package is not standard.